source/documentation/deploying_services/use_a_custom_domain.erb

```diff
@@ -36,12 +34,14 @@ Once you create a CDN service instance, you cannot update or delete the instance
     - `ORG_NAME` is the name of your org
     - `SPACE_NAME` is the name of your space
 
-1. Create the required domain(s) in your organisation. Run the following for each domain:
+1. Create the required domain(s) in your organisation:
 
     ```
-    cf create-domain ORG_NAME DOMAIN_NAME
+    cf create-domain ORG_NAME APEX_DOMAIN_NAME
     ```
 
+    where `APEX_DOMAIN_NAME` is the name of the apex domain, for example `example.com`
```

Comment:
Could we change this to something like `example.service.gov.uk` to make it clear the apex can be a subdomain of an existing domain?